apps/fmr/views/content/en/overview.md

```diff
@@ -1,14 +1,16 @@
-## Before you start
+<h2 class="govuk-heading-m">Before you start</h2>
```

Comment:
does this ensure the correct heading style is used?

Reply:
yes, this would apply additional styles, mainly affecting the margins